### PR TITLE
datetime-fortran 1.4.2 (new formula)

### PR DESCRIPTION
**Pull request by zbeekman**
Fortran library for the manipulation of dates and times.

milancurcic/datetime-fortran#24

CC: @milancurcic

### CODE REVIEW

File: Library/Formula/datetime-fortran.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  head do
+    url "https://github.com/milancurcic/datetime-fortran.git"
+
+    depends_on "autoconf"   => :build
+    depends_on "automake"   => :build
+    depends_on "pkg-config" => :build
+  end
+
+  option "without-test", "Run build time tests"
```
Comment (zbeekman) [inline Library/Formula/datetime-fortran.rb]: I always get confused about phrasing the `"without-..."` descriptions. Should I `s/Run build time tests/Skip build time tests/`?

Comment (DomT4) [inline Library/Formula/datetime-fortran.rb]: Yeah, here it would be `Skip build time tests (Not recommended)` or similar.

Comment (zbeekman) [inline Library/Formula/datetime-fortran.rb]: OK, great. After I fix this, unless you guys have other concerns, this is good to go, in my book.

File: Library/Formula/datetime-fortran.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                          "--disable-silent-rules"
+    system "make", "check" if build.with? "test"
+    system "make", "install"
+    (pkgshare/"test").install "src/tests/datetime_tests.f90"
+  end
+
+  test do
+    ENV.fortran
+    system ENV.fc, "-odatetime_test", "-ldatetime", "-I#{HOMEBREW_PREFIX}/include", pkgshare/"test/datetime_tests.f90"
+    system testpath/"datetime_test"
```
Comment (zbeekman) [inline Library/Formula/datetime-fortran.rb]: non-zero exit status on test failure (with gfortran and ifort, and, AFAIK, all other fortran compilers). This is achieved via `stop <n>` where `0 < n < 256`. The standard doesn't require the `stop` code to be returned from the program as its exit status, but as far as I know, all compilers have adopted this convention. 

(Fun trivia fact, the Fortran standard doesn't even require that the code is on a computer... it just refers to the "processor" as some generic entity)

**Comment by mikemcquaid** [comment]
Thanks again @zbeekman!

**Comment by zbeekman** [comment]
:tada: :bow:

**Pull request closed without merging**